examples/gps.js

```diff
@@ -0,0 +1,46 @@
+// This is an example that uses mineflayer-pathfinder to showcase how simple it is to walk to goals
+// * Requires the npm package "mineflayer-pathfinder"
```

Comment:
Instead of this comment, this could be put in a folder with a package.json, like https://github.com/PrismarineJS/mineflayer/tree/master/examples/viewer